src/net_processing.cpp

```diff
@@ -3828,7 +3828,7 @@ bool PeerLogicValidation::MaybeDiscourageAndDisconnect(CNode& pnode)
     }
 
     // Normal case: Disconnect the peer and discourage all nodes sharing the address
-    LogPrintf("Disconnecting and discouraging peer %d!\n", peer_id);
```

Comment:
> I don't think that is good usability: evidently also long term Core contributors don't follow the described pattern as proven by this PR :)

This line was written by @sipa in b691f2df5f7d443c0c9ee056ab94aa0fc19566d5 and last touched by @jnewbery in 655b1957470c39b.

Reply:
> > > If so then I think it is better is we log when the "alarm should go off" condition is fulfilled: not unconditionally log totally normal events, hope that the user notices that these normal events are "more normal than normally" and that he/she draws the conclusion that something could be wrong.
> >
> > I don't think that is good usability: evidently also long term Core contributors don't follow the described pattern as proven by this PR :)
> 
> This line was written by @sipa in [b691f2d](https://github.com/bitcoin/bitcoin/commit/b691f2df5f7d443c0c9ee056ab94aa0fc19566d5) and last touched by @jnewbery in [655b195](https://github.com/bitcoin/bitcoin/commit/655b1957470c39bcab64917747c9f467444bd809).

My point was simply that Core contributor hebasto who by definition is a power-user didn't interpret the "alarm should go off" logging as such in this case (and TBH I'm not certain he _should_ have). I think it is unreasonable to expect that ordinary users should do that interpretation even if we assume this would be a "alarm should go off" situation.

Reply:
Ok. It's hard to know--I think I'd be more worried about network topology than a peer causing lots of logging, but I'm not sure if that POV is shared.

I'm thinking of looking at the `NET` logging to see if we should consider creating a "high level" net logging category (or a "user level" one) as opposed to the low-level one with all of the invs and other hundreds/thousands of messages per minute. There may be objections to that which I'm not aware of, though.

Reply:
Ok. It's hard to know--I *think* I'd be more worried about network topology here than a peer causing lots of logging, but I'm not sure if that POV is correct or shared.

I'm thinking of looking at the `NET` logging to see if we should consider creating a "high level" net logging category (or a "user level" one) as opposed to the low-level one with all of the invs and other hundreds/thousands of messages per minute. There may be objections to that which I'm not aware of, though.